### PR TITLE
Oded/add prefix play icon

### PR DESCRIPTION
**Pull request by odeds**

### CODE REVIEW

**Comment by odeds** [comment]
jenkins-test

**Comment by odeds** [comment]
jenkins-test

File: src/components/znkAudioPlayer/svg/play-icon.svg [hunk 1]
```diff
@@ -7,8 +7,8 @@
         }
     </style>
 <style type="text/css">
-	.st0{fill:none;stroke:#231F20;stroke-width:3;stroke-miterlimit:10;}
-	.st1{fill:#231F20;}
+	.play-icon .st0{fill:none;stroke:#231F20;stroke-width:3;stroke-miterlimit:10;}
```
Comment (igormas) [inline src/components/znkAudioPlayer/svg/play-icon.svg]: please change to znk-audio-player-play-svg

Comment (odeds) [inline src/components/znkAudioPlayer/svg/play-icon.svg]: changed

**Pull request merged**